client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.html

```diff
@@ -432,7 +432,13 @@
                 <!-- point of order -->
                 <button mat-menu-item (click)="onPointOfOrderButton(speaker)" *ngIf="enablePointOfOrderButton(speaker)">
                     <mat-icon>warning</mat-icon>
-                    <span>{{ speaker.point_of_order ? 'Remove point of order' : ('Point of order' | translate) }}</span>
+                    <span>
+                        {{
+                            speaker.point_of_order
+                                ? ('Remove point of order' | translate)
+                                : ('Point of order' | translate)
+                        }}
```

Comment:
```suggestion
                        {{
                            (speaker.point_of_order ? 'Remove point of order' : 'Point of order') | translate
                        }}
```
Also, this would probably cleaner (saves one pipe)